src/pages/view-test-result/cat-a-mod2/components/debrief-card/debrief-card.ts

```diff
@@ -78,41 +62,37 @@ export class DebriefCardComponent {
   }
 
   public getDrivingFaults(): FaultSummary[] {
-    return this.faultSummaryProvider.getDrivingFaultsList(this.data, TestCategory.BE);
+    return this.faultSummaryProvider.getDrivingFaultsList(this.data, TestCategory.EUAM1);
   }
 
   public getSeriousFaults(): FaultSummary[] {
-    return this.faultSummaryProvider.getSeriousFaultsList(this.data, TestCategory.BE);
+    return this.faultSummaryProvider.getSeriousFaultsList(this.data, TestCategory.EUAM1);
   }
 
   public getDangerousFaults(): FaultSummary[] {
-    return this.faultSummaryProvider.getDangerousFaultsList(this.data, TestCategory.BE);
+    return this.faultSummaryProvider.getDangerousFaultsList(this.data, TestCategory.EUAM1);
```

Comment:
I would prefer is these passed the category in instead of hard coding it. It won't make much difference at the moment but if one of the Mod 2 tests change then it will start causing issues all this hard coding

Reply:
Agreed - will look to change this.